yarn.config.cjs

```diff
@@ -43,15 +43,10 @@ module.exports = defineConfig({
         Yarn.dependencies({ workspace }),
       );
 
-      // Completely ignore apps workspaces
-      if (isApps) {
-        continue;
-      }
-
```

Comment:
This ignored ever check which may have been a bit too broad. There are checks for `private` packages that we should adhere to